developer/src/server/build.sh

```diff
@@ -128,6 +129,7 @@ function installer_server() {
   pushd "$PRODBUILDTEMP"
   npm install --omit=dev --omit=optional
   # See https://github.com/bubenshchykov/ngrok/issues/254, https://github.com/bubenshchykov/ngrok/pull/255
+  rm -f node_modules/ngrok/bin/ngrok
```

Comment:
> We should download ngrok.exe when the user requests it, rather than embedding in the installer.

How is this part of the issue handled?

Reply:
This is already implemented by Keyman Developer IDE, when configuring ngrok